configure.ac

```diff
@@ -73,21 +111,48 @@ AS_IF([test x"$prefix" = xNONE],
 )
 datadir=${prefix}/masterfiles
 
+
+dnl ######################################################################
+dnl Print summary
+dnl ######################################################################
+
+AC_MSG_RESULT()
+AC_MSG_RESULT(Summary of options:)
+AM_COND_IF(HAVE_CORE,
+    AC_MSG_RESULT(Core directory       -> $core_dir),
+    AC_MSG_RESULT(Core directory       -> not set - tests are disabled)
+)
+AM_COND_IF(HAVE_ENTERPRISE,
+    AC_MSG_RESULT(Enterprise directory -> $enterprise_dir),
+    AC_MSG_RESULT(Enterprise directory -> not set - some tests are disabled)
+)
+AC_MSG_RESULT(Install prefix       -> $prefix)
+AC_MSG_RESULT()
+
+
 dnl ######################################################################
 dnl Generate install target list.
 dnl ######################################################################
 
-AC_MSG_NOTICE([generating install targets])
-MASTERFILES_INSTALL_TARGETS=
-for i in cfe_internal controls inventory lib services sketches/meta update; do
-    MASTERFILES_INSTALL_TARGETS="$MASTERFILES_INSTALL_TARGETS $(find "$srcdir/$i" -name '*.cf' -exec printf '%s ' '{}' ';')"
+AC_MSG_NOTICE([generating makefile targets])
+
+MASTERFILES_TESTS=`cd "$srcdir/tests/acceptance/" && find . -name '*.cf' | xargs`
+AC_SUBST(MASTERFILES_TESTS)
+
+MASTERFILES_INSTALL_TARGETS=`echo $srcdir/*.cf`
+
+for i in cfe_internal controls inventory lib services sketches/meta update
+do
+    MASTERFILES_INSTALL_TARGETS="$MASTERFILES_INSTALL_TARGETS `find "$srcdir/$i" -name '*.cf' | xargs`"
 done
-for i in templates; do
-    MASTERFILES_INSTALL_TARGETS="$MASTERFILES_INSTALL_TARGETS $(find "$srcdir/$i" -name '*.mustache' -exec printf '%s ' '{}' ';')"
+for i in templates
+do
+    MASTERFILES_INSTALL_TARGETS="$MASTERFILES_INSTALL_TARGETS `find "$srcdir/$i" -name '*.mustache' | xargs`"
```

Comment:
Just ``xargs`` alone? I'm confused. What does that do?

Also, why remove the ``$(blah blah)`` syntax everywhere? It's supported by all shells, and recommended AFAIK, because it nests properly.